html/templates/settings-payment-method.html

```diff
@@ -6,6 +15,7 @@
 			<div class="form-row form-row-type--{{type}}">
 				<div class="form-label">
 					<label for="{{id}}">{{label}}</label>
+					{{#if description}}<div class="settings-payment-method-description">{{description}}</div>{{/if}}
```

Comment:
* Mixing inline (`<label>`) and block-level (`<div/>`) elements should be avoided.
* Follow class naming of surrounding elements (e.g `form-description`)